setup.py

```diff
@@ -32,6 +32,8 @@
 
 packages = [
     'oauth2client',
+    'oauth2client.contrib',
```

Comment:
@dhermes, I see, yes, the version downloaded via pip is from Nov 18, and does not include the contrib submodule. 

It is notable, though,  that the current docs on readthedocs defaults (seemingly) to the latest version for documentation (at least when I view it). This version discusses the contrib submodule and is (or at least in my experience, was) the first thing I saw when looking up the documentation for the flask_util for the oauth2client library. 


@jonparrott 
As far as I know, submodules need to be declared. See http://stackoverflow.com/questions/15368054/import-error-on-installed-package-using-setup-py
and https://docs.python.org/2/distutils/examples.html#pure-python-distribution-by-package

The reason the explicit declaration is needed, it seems, is because the contrib module is actually a sub-package, with its own __init__.py. 
An alternative fix, therefore might be to remove the __init__.py  from contrib.